src/components/NavBar.tsx

```diff
@@ -19,7 +19,11 @@ const NavBar: React.FC = () => {
       <Tabs value={false} centered>
         <Box p="5px">
           <Link to="/">
-            <img className={classes.navBarLogo} src={c4clogo} />
+            <img
+              className={classes.navBarLogo}
+              src={c4clogo}
+              alt="Code4Community Logo"
```

Comment:
⛏️  remove logo? honestly not entirely sure what alt does

Reply:
In the case an image load fails it will load the alt tag as text. Also screen readers will read the alt tag which is why I have the "logo" bit.